assets/src/edit-story/components/panels/layer/provider.js

```diff
@@ -26,20 +26,85 @@ import StoryPropTypes from '../../../types';
 import Context from './context';
 import useLayers from './useLayers';
 
+const SCROLL_DISTANCE = 5;
+
 function LayerProvider({ children }) {
   const layers = useLayers();
+  const [hasScrollAbove, setHasScrollAbove] = useState(false);
+  const [hasScrollBelow, setHasScrollBelow] = useState(false);
   const [isReordering, setIsReordering] = useState(false);
+  const [scrollDirection, setScrollDirection] = useState(0);
   const [currentSeparator, setCurrentSeparator] = useState(null);
+  const scrollTarget = useRef(null);
+
+  const setScrollTarget = useCallback(
+    (node) => (scrollTarget.current = node.parentElement),
+    []
+  );
+
+  const updateScrollMarkers = useCallback(() => {
```

Comment:
Use `useBatchedCallback` since this callback is called from an animation frame outside of the React's event context.

Reply:
Fixed in a8adb79f.